src/views/MyProfile/Components/VictoryPromiseDisplay/index.js

```diff
@@ -0,0 +1,340 @@
+import Grid from '@material-ui/core/Grid';
+import React from 'react';
+import CardHeader from '@material-ui/core/CardHeader';
+import Tooltip from '@material-ui/core/Tooltip';
+import { gordonColors } from '../../../../theme';
+import { Polar } from 'react-chartjs-2';
+import victory from '../../../../services/victory';
+import './VictoryPromise.css';
+import Button from '@material-ui/core/Button';
+import './VictoryPromise.css';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+
+export default class VictoryPromiseDisplay extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      labels: [
+        'Christian Character',
+        'Intellectual Maturity',
+        'Lives of Service',
+        'Leadership Worldwide',
+      ],
+      datasets: [
+        {
+          data: [1, 1, 1, 1],
+          backgroundColor: [
+            gordonColors.neutral.lightGray,
+            gordonColors.neutral.lightGray,
+            gordonColors.neutral.lightGray,
+            gordonColors.neutral.lightGray,
+          ],
+        },
+      ],
+      options: '',
+      CC: 0,
+      IM: 0,
+      LS: 0,
+      LW: 0,
+      CCColor: gordonColors.neutral.lightGray,
+      IMColor: gordonColors.neutral.lightGray,
+      LSColor: gordonColors.neutral.lightGray,
+      LWColor: gordonColors.neutral.lightGray,
+      defaultVPMode: true,
+      CC_ON: false,
+      IM_ON: false,
+      LS_ON: false,
+      LW_ON: false,
+    };
+  }
+
+  changeMode = () => {
+    console.log(this.state.defaultVPMode);
+    if (this.state.defaultVPMode) {
+      this.setState({ defaultVPMode: false });
```

Comment:
You should be able to just do "this.setState({ defaultVPMode: !this.state.defaultVPMode) });" Which will always reverse it's state, so it should work without the if statement.